### PR TITLE
CORE-9981 Added property keys accessible via flow context properties

### PR DESCRIPTION
**Pull request by pmck91**
please see linked pr in runtime https://github.com/corda/corda-runtime-os/pull/3397

### CODE REVIEW

File: data/avro-schema/src/main/java/net/corda/data/flow/CpiPropertyKeys.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package net.corda.data.flow;
```
Comment (sprintengine) [inline data/avro-schema/src/main/java/net/corda/data/flow/CpiPropertyKeys.java]: The package sounds right but the module "data/avro-schema" doesn't sound right, sounds more like we should have a module inside "data" called "flow-schema" or something. Wdyt @mnesbit ? These are properties that will be used in flows to get from `flowEngine.getContextProperties(key)`

**Review by conal-smith — DISMISSED** [review]

**Review by conal-smith — COMMENTED** [review]
lgtm

**Review by conal-smith — DISMISSED** [review]

**Review by fowlerrr — DISMISSED** [review]
I'm happy with the platform version keys

**Review by mnesbit — APPROVED** [review]
LGTM

**Pull request merged**